Framework/DataHandling/src/LoadSampleShape.cpp

```diff
@@ -97,5 +92,39 @@ void LoadSampleShape::exec() {
   setProperty("OutputWorkspace", outputWS);
 }
 
+/**
+ * Rotates the Shape by a provided matrix
+ * @param sampleMesh The Shape to rotate
+ * @param inputWS The workspace to get the rotation from
+ * @returns a shared pointer to the newly rotated Shape
+ */
+void LoadSampleShape::rotate(MeshObject &sampleMesh,
```

Comment:
This function does not seem to interact with any class state. Could this be instead be a free function?

Reply:
If I change these to be a free function they should be in an anonymous namespace correct? wouldn't I have to remove the tests on these functions then?

Reply:
No, they may be in the same namespace as the algorithm and exported in the same manner as the algorithm.

Being free functions actually makes them simpler to test in this case.

Reply:
this just seems to cause me to get linker errors when compiling my tests

Reply:
Did you use the correct DLL export on the function declaration?